docs/eventing/samples/kafka/resetoffset/README.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Apache Kafka ResetOffset Example"
+linkTitle: "ResetOffset Example"
+weight: 20
+type: "docs"
+---
+
```

Comment:
We no longer require this frontmatter in docs, so you can replace it with just a H1
e.g.
`# Apache Kafka ResetOffset Example`

Reply:
This can be followed up with any editing / clean up by writers https://github.com/knative/docs/issues/4017